packages/components/src/docs/Migration/ui-surrounding-map.mdx

```diff
@@ -0,0 +1,7 @@
+# Migration Guide: Deprecation of `ui-surrounding-map`
+
+## [`ui-surrounding-map`] is deprecated
+
+The `ui-surrounding-map` component will not be a part of the SDS Library. This component was heavily dependent on the Google Maps API and had often changing requirements.
```

Comment:
```suggestion
The `ui-surrounding-map` component will not be a part of the SDS Library. This component was heavily tied to the Google Maps API, offering little beyond a restyled Google Map at the cost of integrating a large library and significantly increasing our bundle size.

For your map-related needs, use the [Google Maps Style Wizard](https://mapstyle.withgoogle.com/) to create a custom map style and use [`sd-map-marker`](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/docs/components-sd-map-marker--docs) to display markers on the map. A starter guide for the Google Map Style Wizard will be made available in future.
```

@yoezlem @coraliefeil A starter for the wizard was something we aimed for, right?

Reply:
I am not sure if we only wanted to do that for design.. Couldn't find any other information on that.